build/LinuxDistrosNativeInstaller.props

```diff
@@ -3,13 +3,11 @@
     <LinuxDistrosNeedNativeInstaller Include="debian.8-x64">
       <DockerFolder>debian</DockerFolder>
       <InstallerExtension>deb</InstallerExtension>
-      <RepositoryId>$(debian8x64RepositoryId)</RepositoryId>
```

Comment:
These variables are specified in the build definition, right?  I assume they will be removed once this is merged.

Reply:
Yes